src/main/java/main/ProYapper.java

```diff
@@ -0,0 +1,36 @@
+package main;
```

Comment:
I appreciate the separation of different types of classes into different packages! It keeps the project well organized. Good job! :)